test/AspNet.Security.OAuth.Providers.Tests/SuperOffice/SuperOfficeTests.cs

```diff
@@ -40,6 +40,7 @@ protected internal override void RegisterAuthentication(AuthenticationBuilder bu
         }
 
         [Theory]
+        [InlineData(System.Security.Claims.ClaimTypes.NameIdentifier, "johm.demo.smith@superoffice.com")]
```

Comment:
No test for `ClaimTypes.Email`?

Reply:
I didn't think it was necessary, but I'll go ahead and add that now for completeness.